packages/cpp/README.md

```diff
@@ -79,6 +79,28 @@ To get this working, you need to enable clangd's global index using the
             "cpp.clangdArgs": "--background-index"
         }
 
+## Using the clang-tidy linter
+
+Note: This functionality is available when using clangd 9 and later.
+
+You can set the preference 'cpp.clang-tidy' to initiate the linter. When the preference is set, there are two possibilities to pass the checks arguments:
```

Comment:
"there are two possibilities to pass the checks arguments"
replace with something like:
`there are two ways to chose which of its built-in checks cland-tidy will use` ?